src/config.ts

```diff
@@ -36,6 +38,14 @@ You can use multiple options in conjunction, e.g. "garbo nobarf ascend"',
       help: "*EXPERIMENTAL* garbo will sacrifice some optimal behaviors to run quicker. Estimated and actual profits may be less accurate in this mode.",
       default: false,
     }),
+    workshed: Args.custom<Item>(
+      {
+        default: $item`none`,
+        help: "Parses the substring of the item name (e.g dna, mayo, asdon) into the corresponding workshed for garbo to intelligently switch into. Additional accepted aliases: cmc, trainset.",
```

Comment:
Instead of explicitly listing acceptable aliases, consider the `options` parameter

Reply:
Setting options makes it appear as though cmc and trainset are the only 2 options for this argument, unless we want to explicitly list everything

Reply:
![image](https://user-images.githubusercontent.com/98746573/212142093-b1332448-c92d-4fc6-9bc5-7f4135b56f14.png)
This is probably the best I can do